docs/index.md

```diff
@@ -42,12 +42,17 @@ calculator.PoweringUp += Raise.Event();
 
 <div class="highlight">
 <pre>
+
+```text
 ReceivedCallsException : Expected to receive a call matching:
     Add(1, 2)
 Actually received no matching calls.
```

Comment:
**question:** do we need both `<pre />` and code fence (triple back-tick) here?

Reply:
I don't know. Removed